int8 drone_id
crazyflie_trajectory_manager/ConfigTrajectoryFloatPayload payload
---
bool proxy_success
bool result